### ISSUE TITLE
Labelling an unlabled button

### ISSUE THREAD

**Issue by hamsajaisal** [NekoX-Dev/NekoX#608]
The switching camera icon In the  video chat has no label. Screen reader says  only button. It will be highly appreciated if you could label it like dynamically. Eg. We want to switch two rear camera, it should say to rear camera button rather switch camera button. hope I am clear.
